docs/en/revocation-lists.rst

```diff
@@ -200,42 +200,65 @@ The Revocation Endpoint MUST be provided by the Issuer within its Metadata.
 Credential Revocation HTTP Response
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The response MUST be an HTTP Response using the status code set to *204 (No-Content)* in the case of successful revocation. If the Digital Credential could not be found by the Issuer, an HTTP Response with status code *404 (Not Found)* MUST be returned. Otherwise an HTTP error response MUST be provided by the Issuer to the Wallet Instance. This response MUST use *application/json* as the content type and MUST include the following parameters:
+In case of successful revocation, the Issuer MUST return an HTTP Response with status code *204 No Content*.
+Otherwise, an HTTP error response MUST be provided by the Issuer using status codes according to the table below. 
 
-  - *error*. The error code.
-  - *error_description*. Text in human-readable form providing further details to clarify the nature of the error encountered.
-
-Below a non-normative example of an HTTP Response with an error.
-
-.. code::
+To revoke an already revoked Digital Credential will result in a successful response.
 
-  HTTP/1.1 400 Bad Request
-  Content-Type: application/json;charset=UTF-8
-
-  {
-    "error": "invalid_request"
-    "error_description": "The signature of credential_pop JWT is not valid"
-  }
-
-The following HTTP Status Codes and Error Codes MUST be supported:
+The following HTTP Status Codes MUST be supported:
 
 .. list-table:: 
     :widths: 20 20 60
     :header-rows: 1
 
     * - **Status Code**
-      - **Error Code**
+      - **Body**
       - **Description**
+    * - *204 No Content*
+      - 
+      - The Digital Credential has been successfully revoked.
     * - *400 Bad Request*
-      - *invalid_request*
-      - The request is not valid due to the lack or incorrectness of one or more parameters. (:rfc:`6749#section-5.2`).
+      - Error code and description
```

Comment:
is this intended to be here?

if yes, why it is not in the similar errors below?

Reply:
Ok, maybe I understood what you meant. 

The response body is required only for the cases in which it effectively adds information for the user. That is, there's no value in sending "server error" as a payload to a 500 Server Error response.

Also, 5xx errors are not always handled by the application but rather by gateways and proxies, so it may be impractical to customize them. As it brings no value, I'd rather not consider the body.

Reply:
I agree with @balanza , the only status code that makes sense to include a body in is the _400 Bad Request_ status code. Having said that, probably we could remove the body column saying that only the 400 (Bad Request) status Code MUST include a body encoded in ``application/json`` format and MUST contain the following parameters: [...].

@peppelinux  WDYT?

Reply:
no, since we have server_error in RFC 6749
https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1

Reply:
I can add the error payload if we feel it's worth it, it doesn't affect the way the communication works.

Let me just point out that:
* RFC6749 does not design the endpoint to use multiple http codes as error, that's why it needs to differentiate by payload
* In RFC6749, the http response's content type is `application/x-www-form-urlencoded`, we are using `application/json`
* 5xx will usually be generated by systems other than the resource server (ex: load balancers, api gateways, firewalls, etc), so it's probable we will have payload not in the expected format.